src/misclibs.F

```diff
@@ -4285,33 +4103,23 @@ subroutine calcener(nstat,rstat,ruh,rvh,zh,rmh,pi0,th0,rho,ua,va,wa,ppi,tha,   &
  
       integer i,j,k
       double precision :: u,v,w,tmp,qtot,ek,ei,ep,et,le,var,tem
-      double precision, dimension(nk) :: foo1,foo2,foo3,foo4
       double precision :: foo1t,foo2t,foo3t,foo4t
 
       ek=0.0d0
       ei=0.0d0
       ep=0.0d0
       le=0.0d0
-      !$acc data create(foo1,foo2,foo3,foo4)
-#ifdef _B4B
-      !$acc update host(ruh,rvh,rmh,ua,va,wa,qv,ql,qi,rho,vr,pi0,ppi)
-#else
-      !$acc parallel default(present) private(i,j,k,u,v,w,tmp,qtot,tem) &
-      !$acc reduction(+:foo1t,foo2t,foo3t,foo4t,ek,ei,ep,le)
-#endif
       !$omp parallel do default(shared)  &
       !$omp private(i,j,k,u,v,w,tmp,qtot,tem)
-#ifndef _B4B
+      !$acc parallel default(present) private(i,j,k,u,v,w,tmp,qtot,tem) &
+      !$acc reduction(+:foo1t,foo2t,foo3t,foo4t,ek,ei,ep,le)
       !$acc loop gang 
```

Comment:
Do we need to add `reduction(+:ek,ei,ep,le)` here?

Reply:
Yes, we do.